CONTRIBUTING.md

```diff
@@ -121,12 +121,14 @@ See more about TypeORM migrations at official docs [Migrations](https://github.c
 
 ## Pull Requests
 
-- Check how to create a [Pull Request](https://help.github.com/articles/creating-a-pull-request/)
-- Send a pull request to `master` branch
-- Fill template
-- Write a meaningful description
-- Include screenshots and animated GIFs in your pull request whenever possible
-- Add `deploy` label if you want to test your changes
+- Check how to create a [Pull Request](https://help.github.com/articles/creating-a-pull-request/) (PR).
+- PR titles must follow [Conventional Commits](#git-commit-messages).
+- Do not include issue IDs in the PR title.
+- Use GitHub's [Draft PR](https://github.blog/2019-02-14-introducing-draft-pull-requests/) feature instead of using "WIP" (Work In Progress) in the title.
+- Consider adding relevant `area:*` label(s) to your PR.
+- Add the `deploy` label if you want the PR deployed to the staging environment.
```

Comment:
Let's mention that this feature doesn't work in PRs made from forked repos, cause I've seen several cases when people tried and it led to failed PR checks
```suggestion
- Add the `deploy` label if you want the PR deployed to the staging environment.
**NB!** This feature doesn't work if you create PR from RS App repo fork  
```

Reply:
updated. thanks.